src/Microsoft.Health.Fhir.Core/Features/Search/SearchService.cs

```diff
@@ -27,6 +27,9 @@ public abstract class SearchService : ISearchService, IProvideCapability
         private readonly IBundleFactory _bundleFactory;
         private readonly IFhirDataStore _fhirDataStore;
 
+        // Value which is subtracted from Now when querying _history without _before specified
+        private readonly int historyCurrentTimeBufferInSeconds = -3;
```

Comment:
This is a bit of a red flag to me. After reflecting on this a bit, I think we should not try to add `_before` when user does not specify. This number will depend on a number of things that may not be in out control.

Reply:
I have reverted the code that added the _before param when none was specified by the caller.